grafana-reporting/MATCHER-1490071813409-dashboard.json

```diff
@@ -117,7 +117,7 @@
                   "type": "avg"
                 }
               ],
-              "query": "name:All",
+              "query": "name:\"All\"",
```

Comment:
do you know if it's a bad named data or if it's a specific keyword?

Reply:
It is for the `All` matcher.